### PR TITLE
Added description aufgabe 2

### PR DESCRIPTION
**Pull request by Nicke44**
Check german and if it makes sense

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,3 +20,6 @@
 
 #### Erklärung
 
+*Aufgabe 2:* Das Anwendungsfalldiagram zeigt Anwendungsfälle des Autovermietungssystems. Die meisten Fälle werden aus der Perspektive des Servers beschrieben. Nur der Anwendungsfall "Buchung" ist direkt mit dem Kunden verbunden.
```
Comment (dumbdannot) [inline README.md]:
```suggestion
*Aufgabe 2:* Das Anwendungsfalldiagramm zeigt Anwendungsfälle des Autovermietungssystems. Die meisten Fälle werden aus der Perspektive des Servers beschrieben. Nur der Anwendungsfall "Buchung" ist direkt mit dem Kunden verbunden.
```

File: README.md [hunk 2]
```diff
@@ -20,3 +20,6 @@
...
+Eingeschlossenen Anwendungsfälle (Buchung,  Buchungsinformationen und Alter, Persönliche Daten, Zahlungsdaten, usw.) sind notwendig, um Basis-Anwendungsfälle korrekt zu funktionieren (Bestätigungs-E-Mail Erstellung, Kostenvoranschläge Bereitstellung, Buchung in Datenbank Speicherung, usw.).
```
Comment (dumbdannot) [inline README.md]:
```suggestion
Eingeschlossene Anwendungsfälle (Buchung, Buchungsinformationen und Alter, Persönliche Daten, Zahlungsdaten, usw.) sind notwendig, um Basis-Anwendungsfälle korrekt zu funktionieren (Bestätigungs-E-Mail Erstellung, Kostenvoranschläge Bereitstellung, Buchung in Datenbank Speicherung, usw.).
```

**Review by dumbdannot — COMMENTED** [review]
German seems fine to me, just tiny nits.

**Review by dumbdannot — APPROVED** [review]
LGTM, waiting for @MassimiIiano's feedback

**Review by massmola — APPROVED** [review]

**Pull request merged**